core/ui/SideBar/Open.tid

```diff
@@ -23,10 +23,20 @@ $button$
 </$droppable>
 \end
 
+\define open-tab-close-tiddler-button()
+<$button message="tm-close-tiddler" tooltip={{$:/language/Buttons/Close/Hint}} aria-label={{$:/language/Buttons/Close/Caption}} class="tc-btn-invisible tc-btn-mini tc-small-gap-right">{{$:/core/images/close-button}}</$button>
+\end
+
+<!-- Override one or both of the following 2 macros with a global macro of the same name
```

Comment:
Is it actually possible to override a local macro definition with a global macro? I don't think it will work here.

Reply:
You are right. This one doesn't work, since it is tagged: `$:/tags/SideBar` ... which I obviously did confuse with tags: `$:/tags/Macro`